pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -421,20 +421,43 @@ public static class QueryOptionKey {
         public static final String USE_SCAN_REORDER_OPTIMIZATION = "useScanReorderOpt";
         public static final String MAX_EXECUTION_THREADS = "maxExecutionThreads";
 
-        /** Number of groups AggregateOperator should limit result to after sorting.
-         *  Trimming happens only when (sub)query contains order by and limit clause. */
+        /**
```

Comment:
There are quite some unrelated javadoc format changes. I guess you should turn off the IDE auto-reformat javadoc

Reply:
done